x/permission/keeper/migrate.go

```diff
@@ -0,0 +1,35 @@
+package keeper
+
+import (
+	"fmt"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/axelarnetwork/axelar-core/x/permission/exported"
+)
+
+// GetMigrationHandler returns a migration handler from v1 to v2
+// The order of role enums has changed, so we have to migrate all government accounts.
+// Enums are int32 underneath, so without this migration, the meaning of the value would change and open up the system to attacks
+func GetMigrationHandler(k Keeper) func(sdk.Context) error {
+	roleMapping := map[exported.Role]exported.Role{
+		0: exported.ROLE_UNRESTRICTED,
```

Comment:
Why do the migration? Can't we just add the Unrestricted role at the end of the enum list?